disablestrongnamevalidation.ps1

```diff
@@ -0,0 +1,2 @@
+& "C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.6.2 Tools\x64\sn.exe" -Vr *,31bf3856ad364e35
+& "C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.6.2 Tools\sn.exe" -Vr *,31bf3856ad364e35
```

Comment:
It's the same tool compiled for two different platforms.
Are we running the tests on a 32 bit system?
Is the 64 bit version erroring out but the script continues?